src/templates/ja/2020/methodology.html

```diff
@@ -0,0 +1,426 @@
+
+      <p>
+        <a href="https://github.com/reworkcss/css">Rework CSS</a>はJavaScriptベースのCSSパーサーです。スタイルシート全体を受け取り、個々のスタイルルール、セレクタ、ディレクティブ、値を区別するJSONエンコードされたオブジェクトを生成します。
+      </p>
+
+      <p>
+        この特別な目的のツールは、<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>章の多くのメトリクスの精度を大幅に向上させました。各ページのすべての外部スタイルシートとインラインスタイルブロックのCSSを解析し、解析可能な状態にするために問い合わせを行いました。BigQueryのHTTP Archiveデータセットとの統合方法については、<a href="https://discuss.httparchive.org/t/analyzing-stylesheets-with-a-js-based-parser/1683">このスレッド</a>を参照してください。
+      </p>
+    </section>
+
+    <section>
+      <h2 id="process"><a href="#process" class="anchor-link">分析プロセス</a></h2>
+
+      <p>
+        Web Almanacの計画と実行には、Webコミュニティの数十人の貢献者の協力を得て、約1年を要しました。このセクションでは、Web Almanacに掲載されているメトリクスを選択した理由、それらがどのようにして照会されたか、そして解釈されたかについて説明します。
+      </p>
+    </section>
+
+    <!-- To be translated <section>
+      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>
+
+      <p>
+        The 2020 Web Almanac kicked off in June 2020 as a <a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">request for new authors</a>. After a round of brainstorming, 22 chapters were solidified and we began <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">assigning subject matter experts and peer reviewers to each chapter</a>. This process had some inherent bias because of the challenge of getting volunteers to commit to a project of this scale. Thus, many of the contributors are members of the same professional circles. One explicit goal for future editions of the Web Almanac is to encourage even more inclusion of underrepresented and heterogeneous voices as authors and peer reviewers.
+      </p>
+
+      <p>
+        We spent July 2020 pairing people with chapters and getting their input to finalize the individual metrics that will make up each chapter.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analysis</a></h3>
+
+      <p>
+        In July and August 2020, with the stable list of metrics and chapters, data analysts triaged the metrics for feasibility. In some cases, <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">custom metrics</a> needed to be created to fill gaps in our analytic capabilities.
+      </p>
+
+      <p>
+        Throughout August 2020, the HTTP Archive data pipeline crawled several million websites, gathering the metadata to be used in the Web Almanac.
+      </p>
+
+      <p>
+        The data analysts began writing queries to extract the results for each metric. In total, <strong>TBC queries</strong> were written by hand! You can browse all of the queries by chapter in the <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> directory of the project's GitHub repository.
+      </p>
+    </section>-->
```

Comment:
Can a native speaker translate this whole section and remove the `<!--` and `-->` marks?

Reply:
Yes, I think can.

Reply:
```suggestion
    <section>
      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>

      <p>
        2020年Web Almanacは、2020年6月に<a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">新規執筆者を募集して</a>スタートしました。ブレーンストーミングのラウンドの後、22の章が固まり、私たちは<a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">各章に主題専門家とレビュワーを割り当てる作業を開始しました</a>。このプロセスには、この規模のプロジェクトにコミットするためにボランティアを集めるという課題があったため、いくつかの固有の偏りがありました。したがって、寄稿者の多くは同じ専門家サークルのメンバーです。Web Almanacの今後のエディションの明確な目標の1つは、著者やレビュワーとして、代表性の低い、異質な声をより多く取り入れることを奨励することです。
      </p>

      <p>
        私たちは2020年7月に、人々と支部のペアリングを行い、各支部を構成する個々の指標を最終的に決定するために、彼らの意見を聞きました。
      </p>
    </section>

    <section>
      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analysis</a></h3>

      <p>
        2020年7月と8月に、メトリクスとチャプターの安定したリストを持って、データアナリストは、実現可能性のためにメトリクスをトリアージしました。場合によっては、分析能力のギャップを埋めるために、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>を作成する必要がありました。
      </p>

      <p>
        2020年8月を通して、HTTPアーカイブデータパイプラインは数百万のウェブサイトをクロールし、Web Almanacで使用するためのメタデータを収集しました。
      </p>

      <p>
        データアナリストは、各メトリクスの結果を抽出するためのクエリを書き始めました。合計すると、<strong>TBC クエリ</strong>が手書きで書かれていました! プロジェクトのGitHubリポジトリの <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> ディレクトリで、章ごとにすべてのクエリを閲覧できます。
      </p>
    </section>
```